posts/2017-05-10-serverless-v1.13.0.md

```diff
@@ -0,0 +1,157 @@
+---
+title: New in Serverless v1.13 - v1.13 opens up world of configuration possibilities with new async variable support
```

Comment:
"Async variable support added for easier configuration & secret management - Serverless v1.13"
 
or "Async variable support added in Serverless v1.13 for better configuration & secret management"

or "Async variable support added in Serverless v1.13 for enhanced configuration & secret management" <----- My faves